pages/getting-started/typography/typography.html

```diff
@@ -41,5 +41,6 @@ <h6>h6. UI Game Heading</h6>
 		</div>
 
 		<script src="/prism/prism.js"></script>
+		<script src="/app.js"></script>
```

Comment:
You can script inside head tag by using defer keyword which will only load after the DOM is completely rendered

Reply:
okay